docs/src/index.md

```diff
@@ -71,8 +73,9 @@ julia> dx
   200.0
 ```
 
-Both the inplace and "normal" variant return the gradient. The difference is that with
-[`Active`](@ref) the gradient is returned and with [`Duplicated`](@ref) the gradient is accumulated in place.
+Both the inplace and "normal" variant return the gradient. The difference is that with [`Active`](@ref) 
```

Comment:
This wording seems weird. The inplace version returns `((nothing,),)`, it's written right there. And inplace / "normal" are new terms here, which aren't the terms you need to learn to understand Enzyme.

> The version with `Active` (for immutable inputs like x::Float64) returns the gradient. The version with `Duplicated` (for mutable inputs like `x::Vector{Float64}`) instead writes the gradient into the `Duplicated` object, and returns `nothing` in the corresponding slot of the returned `derivs`.

Reply:
```suggestion
Both versions calculate the same derivatives. The difference is that with [`Active`](@ref) arguments
```